### DISCUSSION TITLE
FPP / FalconPlayer

### DISCUSSION THREAD

**Discussion by rdrgmnzs** [selfhst/icons#136] [New Icon Request]
Would love to have access to the redesigned Falcon Player logo / icon.

Icon: https://github.com/FalconChristmas/fpp/blob/master/www/images/redesign/app-icon.svg
repo: https://github.com/FalconChristmas/fpp
Wiki: https://falconchristmas.github.io/

**Comment by shollyethan** [comment]
Falcon Player has been added - thanks!
